week2/homework/src/index.js

```diff
@@ -1,3 +1,95 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const fs = require('fs');
+let command = process.argv[2];
+let value = process.argv[3];
+let listFile = fs.readFileSync('./data.json', 'utf8');
+let listArr = JSON.parse(listFile);
+
+switch (command) {
+  case 'list':
+    viewTasks();
+    break;
+  case 'add':
+    addTask();
+    break;
+  case 'help':
+  case undefined:
```

Comment:
`case undefined` is the same as `default`. you should move `case help` all the way down before `default` as the assignment says you should return the `help` section if you call the app without any command :)